openapi/generated/api_backend.yaml

```diff
@@ -2218,6 +2192,12 @@ definitions:
     allOf:
       - $ref: '#/definitions/CreatedMessageWithContent'
       - $ref: '#/definitions/EnrichedMessage'
+  Ulid:
+    type: string
+    description: Ulid string.
+    format: Ulid
+    x-import: '@pagopa/ts-commons/lib/strings'
```

Comment:
Possible improvement: instead of importing a custom decoder, just create a _PatternString_ from an explicit [OA3 pattern](https://swagger.io/docs/specification/data-models/data-types/#pattern)

(don't remember if the generator already handles it)

Reply:
Actually this import could be useless because for parameters we can't use definitions.
The parameter uses a PatternString with the same Ulid pattern, then inside the src we use the Ulid decoder to validate.
A client would generate the Ulid module, but the client code would be using PatternString.
Do you think we can remove it from the definitions?